pymtl3/passes/backends/sverilog/import_/test/ImportedObject_test.py

```diff
@@ -55,6 +55,36 @@ def construct( s ):
   a._tv_out = tv_out
   do_test( a )
 
+def test_reg_incomplete_portmap( do_test ):
+  def tv_in( m, test_vector ):
+    m.in_ = Bits32( test_vector[0] )
+  def tv_out( m, test_vector ):
+    if test_vector[1] != '*':
+      assert m.out == Bits32( test_vector[1] )
+  class VReg( Component ):
+    def construct( s ):
+      s.in_ = InPort( Bits32 )
+      s.out = OutPort( Bits32 )
+      s.config_sverilog_import = ImportConfigs(
+          vl_src = get_dir(__file__)+'VReg.sv',
+          port_map = {
```

Comment:
I wonder if we should use the port object s.in_ instead of name in_

Reply:
We could. They have about the same number of characters so I'm wondering what would be the benefit of using port objects instead of names...

Reply:
it could throw better errors earlier when the user specifies the key using an actual object i guess?

Reply:
We can do this later this month